### DISCUSSION TITLE
What's going on with Windows Update (KB5025239)?

### DISCUSSION THREAD

**Discussion by passwordnotavailable** [valinet/ExplorerPatcher#1306] [Q&A]
I may be way out of my depth here, but I just installed (KB5025239) and first of all Classic Shell no longer worked, so I downloaded and installed Open Shell, that now works. However, Explorer Patcher wouldn't work and I disovered it was an old version, so uninstalled it and then installed the latest Explorer Patcher from here. That now works, but I am wondering if there is anything missing from it? I thought that I saw new toolbar on Explorer Patcher, but I definitely don't have it. I hate this stupid new Windows update, I have lost my quick launch, which I did have on W11 before this update, I can't make the icons smaller now and I am not able to lock or unlock the taskbar or make the taskbar double as I had it before. One of the programs which has always started up on boot, does not now start and I can't add things like the download folder to my taskbar. Perhaps it is me, but any help would be appreciated please and thanks.

Update: I have just found my Quick Launch folder, however I am unable to add anything from it or even the whole folder to the taskbar.

**Comment by mistertheloaf** [comment]
Things were OK until KB5023778  that was released on March 28.  Then Microsoft changed something in Explorer.exe

Some people have reported that having ExplorerPatcher download new symbols -- and a few reboots -- fixes the problem, but I haven't been able to get it to work yet.

ExplorerPatcher is a very good program and hopefully the author will release an updated version that works properly with the latest Windows updates.   BUT, patching explorer.exe is **_very_** tricky.  Every time Microsoft puts out a new Windows update  there is a chance that it might break something.

**Reply by passwordnotavailable** [reply]
I have the enable symbol downloads checkmarked and have restarted  File Explorer, but nothing has changed. I have now pinned my quick launch folder to the top of the menu on Open Shell, which is OK, but I would still prefer to be able to put the quick launch back on my taskbar and to be able to make the icons smaller, but to no avail. Perhaps, (hopefully) the developer of ep might be able to come up with something to get things back to as they were before these Windows updates?

**Reply by tallman69** [reply]
Since you uninstalled and reinstalled EP you may need to reactivate the Quick Launch toolbar just like you did when you first installed EP.  Have you tried doing that?

**Reply by passwordnotavailable** [reply]
Yes, I have tried doing that. However with the latest Windows update quick launch is just not available anymore.

**Reply by tallman69** [reply]
I have the latest windows updates on my two Windows 11 computers along with the most recent version of EP and the Quick Launch toolbar works perfectly for me.  Something may have gotten corrupted along the way.  Have you scanned for malware and also run sfc?

**Reply by Azghul** [reply]
Is the Version to download the newest version or is there some update possible?

**Reply by tallman69** [reply]
> Is the Version to download the newest version or is there some update possible?

When you download the version from this website it is the latest version.  There is no update needed.

**Reply by tallman69** [reply]
@passwordnotavailable
 
Are you using the Language Bar?  If so you may find this thread helpful.  

https://answers.microsoft.com/en-us/windows/forum/all/windows-10-quick-launch-toolbar-disappears-on/193a4890-9413-41a4-abea-f56688ba5eeb

Apparently, the disappearing Quick Launch toolbar has been an issue starting with Windows 10, and for some people it seems to be related to the Language Bar placement.

**Reply by tallman69** [reply]
@passwordnotavailable 

I also discovered from the above posted thread that you can put your Quick Launch folder any place you like and name the folder whatever you want.  Then create a new toolbar directed to that folder and customize the toolbar as desired.  I tried putting the folder on my data drive and it worked perfectly.  Maybe that will help you.

**Reply by passwordnotavailable** [reply]
@tallman69 

Thanks for your help. I actually uninstalled (KB5025239) but that did not make any difference, so I then clicked restore default settings in EP and that brought everything back and set EP to Windows 10 by default. Quick Launch returned and a lot of settings on EP appeared that were not there before. Not sure why it worked, but it did. I'm not sure what will happen if (KB5025239) automatically downloads and installs again, so I have put my downloads on pause for the time being. 

P.S. I do have the language bar in my taskbar it's still there and always has been, so I'm not sure if that is an issue, but you never can tell and if everything goes haywire again, I'll start with removing that.

Regarding the Quick Launch Folder, I couldn't create a new toolbar because the option wasn't there before, but it is there now after I reset the default settings in EP. My QL came back on it's own after I did that.

**Reply by tallman69** [reply]
@passwordnotavailable 

Good to hear the good news!  Hopefully all will go well when KB5025239 gets reinstalled.  It is a chance we take, by using third party software like EP and OpenShell that alter the Windows interface, that MS will break EP and OpenShell when they make changes to Windows.  But I sure like having those two programs to help make Windows 11 less of a learning curve to use.

**Reply by passwordnotavailable** [reply]
@tallman69 

Time will tell I guess. Yes, why Microsoft have to keep changing things just when you get used to how something works, is beyond me. Some guy sitting in an office trying to dream up what else he can do to p**s off Windows users, comes to mind. :)

**Comment by Azghul** [comment]
Hi all,

i have the "black screen" issue with the new Win11 Patch. I think, i just need to wait for an updated version. Still hate the new taskbar from Win11....

**Reply by topia** [reply]
What is the black screen issue? There are some problems reported in this forum, for example, for the weather widget and for Moden Apps in the Win10 Start menu, but I don't see that. if you mentioned repeated crashing, check the relevant issue for the fix.

**Reply by Azghul** [reply]
like the same reported here:

https://github.com/valinet/ExplorerPatcher/discussions/869

**Reply by topia** [reply]
I didn't notice it due to being older than one year. But some replies treated it as a crash loop problem.
But I doubt it because the latest ExplorerPatcher (22621.1413.54.5) already implemented the measures for the crash loop.

**Reply by HazardyKnusperkeks** [reply]
I had the same issue and uninstalled the update.

**Reply by Azghul** [reply]
> I didn't notice it due to being older than one year. But some replies treated it as a crash loop problem. But I doubt it because the latest ExplorerPatcher (22621.1413.54.5) already implemented the measures for the crash loop.

Well, can't tell you about the details. 
If i uninstall the Windows Update, EP is working.

**Reply by topia** [reply]
Hmm, but I think the developer can't fix your problem without the details; unfortunately, applying updates is a one-time event and it might be difficult to reproduce.

**Reply by Azghul** [reply]
At least it is reproducable for me :)
Seems to be a problem with the explorer.exe. Since nothing is starting. We will see. Either there might a patch (if other people have the same issue and might be able to give more detailed information) or i have to deal with the new Win11 stuff.

**Comment by Pawloland** [comment]
It's funny, MS officially recognizes EP in the [release note for KB5025239](https://support.microsoft.com/en-gb/topic/april-11-2023-kb5025239-os-build-22621-1555-5eaaaf42-bc4d-4881-8d38-97e0082a6982). Apparently so many users avoid new start menu and taskbar with EP, that folks at MS couldn't ignore it any longer and decided to explicitly acknowledge the issue, instead of fixing the regressions of the new UI.
![obraz](https://user-images.githubusercontent.com/59684145/233640557-00a7d92d-0af9-4915-9f0e-9360219c1c86.png)

**Reply by passwordnotavailable** [reply]
@Pawloland 

ExplorerPatcher have released a version that lists this issue as resolved????? Have they? The latest version I have is 22621.1413.54.5 is this the latest version? If so, this version has not resolved the issue with KB5025239. Well not on any of my computers anyways.

**Reply by Pawloland** [reply]
I know as much as you. I am in even worse situation actually, because this KB5025239 always fails to install, and on my current OS version, the latest 22621.1413.54.5 EP is broken 🙃. I wanted to install the latest windows update, because EP is supposedly working perfectly with it. Therefore I can't answer your question, because I didn't even get to try it.

**Reply by tallman69** [reply]
>ExplorerPatcher have released a version that lists this issue as resolved????? Have they? The latest version I have is 22621.1413.54.5 is this the latest version? If so, this version has not resolved the issue with KB5025239. Well not on any of my computers anyways.

22621.1413.54.5 is the latest version.  

I have two Windows 11 22H2 computers both running the latest version of EP, and I have no issues with KB5025239.  Have you tried uninstalling EP and then installing KB5025239 then reinstalling EP?

**Reply by passwordnotavailable** [reply]
@tallman69 

I have 3 Windows 11 22H2 computers myself. I have checked one of them and the 22H2 update and the KB5025239 is showing as installed on the 18th April, so 4 days ago. Everything on this particular computer is working fine. I have the quick launch and EP is also working good.

My main computer had the Windows 11 22H2 update installed on 28th September, 2022 and I'm currently downloading and installing the KB5025239 update which I had previosuly uninstalled on this computer which messed everything up, so we will see what happens this time when it has finished installing. If it goes worng again, I will do as you have suggested above.

My other computer is a Windows 11 Pro, I haven't checked that one yet, but the quick launch and EP is working fine, so I will need to get into it later and check to see if KB5025239 is installed or not.

I'll update later and we can see if  all my problems are solved......or not!

**Reply by passwordnotavailable** [reply]
@tallman69 

Just a quick update, the KB5025239 has just finished installing on this, (my main) computer nd everything is good and I still have quick launch and my EP is working normally too. The only thing I can think of is that in some way EP was corrupted when I first installed it. I'll update on my 3rd computer later.
